docs/remote-devcontainer.md

```diff
@@ -0,0 +1,47 @@
+# Contributing with VS Code Remote Container Extension
+
+ We have laid out steps for contributing to the **Azure Key Vault Provider for Secrets Store CSI Driver** using the `VS Code - Remote Container Extension`.
+
+## Prerequisites
+1. Azure Subscription
+
+## Fork and Clone Repository
+
+Before we dive into setting up a remote container environment, fork and clone the repository first. Once cloned, enter into the `root` folder of the project:
+
+  ```bash
+  $ cd secrets-store-csi-driver-provider-azure
+  ```
+
+## VS Code with Remote Container Extension
+
+The [VS Code Remote Container Extension](https://code.visualstudio.com/docs/remote/containers) utilizes the `.devcontainer` folder to build a remote container that will have all necessary dependencies installed to contribute to the **Azure Key Vault Provider for Secrets Store CSI Driver**.
+
+### Dependencies Included Inside The Remote Container
+
+- `yq and jq` command line utilities for manipulating YAML and JSON files
+- `Azure CLI` for access to your Azure Subscription
+  - Your `.azure` folder on your host machine is mounted into the container, so you will be logged in to the same Azure Subscription.
+- `kind` for to allow configuring and using KinD clusters
```

Comment:
```suggestion
- `kind` to configure and use [KinD](https://github.com/kubernetes-sigs/kind) clusters
```

Reply:
@bdlb77  can you PTAL at this suggestion?